scripts/postinstall.js

```diff
@@ -9,23 +8,14 @@ if (process.env.INIT_CWD) {
 
 const pwd = process.cwd()
 const nodeModulesFolder = path.join(pwd, 'node_modules')
-const from = path.resolve(nodeModulesFolder, 'nexus-plugin-prisma', 'global-type.d.ts')
+const from = path.resolve(nodeModulesFolder, 'nexus-plugin-prisma/global-type.d.ts')
 
 // Prevent from crashing if the typings can't be found for whatever reason
-if (jetpack.exists(from)) {
-  const destDir = path.resolve(nodeModulesFolder, '@types', 'nexus-plugin-prisma')
+if (fs.exists(from)) {
+  const destDir = path.resolve(nodeModulesFolder, '@types/nexus-plugin-prisma')
 
-  jetpack.dir(destDir)
-  jetpack.copy(from, path.join(destDir, 'index.d.ts'), { overwrite: true })
+  fs.dir(destDir)
+  fs.copy(from, path.join(destDir, 'index.d.ts'), { overwrite: true })
 }
 
-console.log(chalk.bold.yellowBright('----------------------------------'))
```

Comment:
Don't think this is very helpful anymore and even if we wanted to keep it, would need rethinking now that the packages are merged.

No?